Proposals/xgov-37.md

```diff
@@ -0,0 +1,118 @@
+company_name: Folks Finance
+category: dApps
+focus_area: Defi,NFT
+open_source: Yes
+amount_requested: 985000
+status: Final
+---
+
+## Abstract
+
+The ALGO Amount requested refers to the first milestone of the entire project, the plan is to propose a single milestone during each quarter so to give opportunity for more projects to request grants in each single xGov Grant batch.
+
+In the context of this proposal, the definition of “NFTs” encompasses any on-chain tokenized asset. This includes, but is not limited to:
+
+- Tokenized Real-World Assets
+- Lofty property tokens
+- Vesta Equity property tokens
+- NFTs incl. art, collectibles and in-game assets
+- NFDs
+- Opulous MFTs
+
+To truly spark activity in the ecosystem and unlock new opportunities, NFTs must be financialized. The proposal specifically focuses on including NFTs in lending and borrowing operations.
+
+NFTs are one of the most compelling use cases of the Algorand blockchain. They are heavily traded and exchanged, and the community which supports them is strong and vibrant. Because of Algorand’s ASA technology, tokenizing assets is incredibly simple and powerful in this ecosystem. However, financial tooling for NFTs barely exists on Algorand at the moment.
+
+This xGov proposal leads to the development of a decentralized NFT lending platform where users can collateralize their NFTs to borrow fungible tokens.
+The platform features two markets: a peer-to-peer market and a peer-to-pool market, offering flexibility and liquidity to participants.
+
+The project’s aim is to design and develop the necessary infrastructure, protocols, and algorithms for the NFT lending platform. Extensive market research, rigorous testing, and optimization efforts will be undertaken to deliver a user-friendly platform with robust security measures.
+
+The development of this NFT lending platform will evolve the ecosystem by unlocking the value of tokenized assets and enabling access to liquidity through borrowing fungible tokens. The tools will be open-sourced, meaning any project can integrate this technology and shape it to their needs. Benefits and revenue will proliferate around Algorand’s network. NFT Lending fosters financial inclusion, empowers NFT holders to leverage their assets, and opens up new opportunities for participants within the tokenization space. The long-term applications of this technology are limited only by how much value, and how many different things, can be tokenized.
+
+## Team
+
+[Benedetto Biondi](https://www.linkedin.com/in/benedetto-biondi-030248199/) is the co-founder and CEO of Folks Finance, one of the leading projects on Algorand, offering a full set of DeFi tools for managing Digital assets. Benedetto has accrued 3 years of experience as a Blockchain Project Manager, is a mentor in the Algorand Miami and Europe Accelerators, a DeFi advisor at Climatecoin, and a DeFi lecturer at the course of Economics and Law for Digital Assets at the University of Florence.
```

Comment:
External link needs to be formatted using html <a>
eg `<a href="https://www.linkedin.com/in/benedetto-biondi-030248199/">Benedetto Biondi</a>`